js/mail.js

```diff
@@ -1196,6 +1187,35 @@ var Mail = {
 $(document).ready(function() {
 	Mail.UI.initializeInterface();
 
+	// Enables infinite scroll ability.
+	setInterval(function() {
+
+		// Element identifies the element we want to trigger our loading on.
+		//var element = $('.mail_message_summary:nth-last-child(1)');
+		var element = $('#load-more-mail-messages');
+		var outerHeight = element.outerHeight();
+		var offset = element.offset();
+		var scrollLength = $('#mail_messages').scrollTop();
+		var screenHeight = $(document).height();
+		var messageListHeight = scrollLength + offset.top + outerHeight;
+
+		if ((scrollLength + screenHeight) >= messageListHeight) {
+			Mail.UI.messageView.loadMessages('addMore');
+		}
+
+	}, 1000);
```

Comment:
You don't include another library to solve a simple problem. If you need to stop the timer, set a bind. When the window is out of focus, stop timer. I was planning on doing this as well, even though the performance hit is so negligible, hence waiting. I wonder why you're not making a new issue of it, since I'm not doing anything that isn't *already* done, even by you nonetheless.